src/components/Gallery.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import '../css/gallery.css';
+
+const Gallery = () => {
+  const items = [
+    {
+      category: "exchange",
+      title: "Digital Exchanges",
+      description: "Secure, high-performance trading platforms",
+      image: "/api/placeholder/600/400"
+    },

```

Comment:
another good candidate to store on db and call with API at a later date, rarther than hardcode, its fine for now though